scripts/test-truncating-of-client-hello.py

```diff
@@ -35,6 +34,8 @@ def help_msg():
     print("                names and not all of them, e.g \"sanity\"")
     print(" -e probe-name  exclude the probe from the list of the ones run")
     print("                may be specified multiple times")
+    print(" -n num         only run `num` random tests instead of a full set")
+    print("                (excluding \"sanity\" tests)")
```

Comment:
Not sure if this description is clear enough. The "(excluding "sanity" tests)" in this part of the sentence might mean either that the number does not include the sanity tests or that the sanity tests are excluded from the full set. I think the meaning is the former, so I think it should say something like
```only run `num` (excluding "sanity" tests) random tests instead of a full set```
or
```only run `num` random tests instead of a full set ("sanity" tests are always executed)```

Reply:
the latter one does sound better, updated